.github/workflows/reusable_pull_request.yml

```diff
@@ -3,18 +3,26 @@ name: Pull Request
 on:
   workflow_call:
     inputs:
-      enable_unit_tests:
+      unit_tests_enabled:
         type: boolean
         description: "Boolean to enable the unit tests job. Defaults to true."
         default: true
-      enable_api_breakage_check:
+      api_breakage_check_enabled:
         type: boolean
         description: "Boolean to enable the API breakage check job. Defaults to true."
         default: true
-      enable_docs_check:
+      docs_check_enabled:
         type: boolean
         description: "Boolean to enable the docs check job. Defaults to true."
         default: true
+      acceptable_language_check_enabled:
+        type: boolean
+        description: "Boolean to enable the acceptable language check job. Defaults to true."
+        default: true
+      acceptable_language_check_word_list:
+        type: string
+        description: "List of unacceptable words. Defaults to a sensitive list of words."
```

Comment:
... a sensitive list of words? I'm not sure the list of words has feelings.

Reply:
Yeah, maybe just remove this, the default is just below. Presumably any tooling that's showing the action can render the default if it wants from the YAML.